src/core_ct/core.py

```diff
@@ -281,3 +281,63 @@ def chunk(self, x1=0, y1=0, z1=0, x2=None, y2=None, z2=None) -> Core:
 
         new_core = Core(self.pixel_array[x1:x2, y1:y2, z1:z2], self.pixel_dimensions)
         return new_core
+
+    def shape(self) -> tuple[int, int, int]:
+        """
+        Get the pixel dimensions of the core scan.
+
+        Arguments:
+        ---------
+            none
+
+        Returns:
+        -------
+            The pixel dimensions of the core scan.
+        """
+        return self.pixel_array.shape
+
+    def dimensions(self) -> tuple[float, float, float]:
+        """
+        Get the dimensions of the scan in mm.
+
+        Arguments:
+        ---------
+            none
+
+        Returns:
+        -------
+            The dimensions of the scan in mm.
+        """
+        return tuple(
+            size * dimension
+            for size, dimension in zip(self.pixel_array.shape, self.pixel_dimensions)
+        )
+
+    def volume(
+        self, min_density: float = 0, max_density: float = float("inf")
```

Comment:
Also negative brightness values are valid, so the `min_density` default should be negative infinity

Reply:
Talked about this in person. We decided on removing the `min_density` and `max_density` arguments and making this function reliant on the filter method.